app/src/main/res/values/strings.xml

```diff
@@ -175,27 +175,27 @@
     <string name="version_name_title">Version name:</string>
     <string name="backup_mode_title">Backup mode:</string>
     <string name="last_backup_title">Last backup:</string>
-    <string name="app_s_type_title">App\'s type:</string>
+    <string name="app_s_type_title">App type:</string>
     <string name="encrypted_data_title">Encrypted data:</string>
     <string name="other_filters_options">Special filters:</string>
-    <string name="deleted_backup">Deleted the backup</string>
-    <string name="empty_filtered_list">The filtered List is empty, therefore setting the filters back to default</string>
+    <string name="deleted_backup">Backup deleted</string>
+    <string name="empty_filtered_list">Resetting the filters because the filtered list is empty.</string>
     <string name="ignore_battery_optimization_title">Battery Optimization</string>
-    <string name="ignore_battery_optimization_message">Backups could stop sometimes. To prevent this, you should turn off the battery optimization</string>
-    <string name="ignore_battery_optimization_not_supported">This device doesn\'t support ignoring battery optimizations unfortunately!</string>
```

Comment:
Adding an exclamation mark is rude in many languages, where "unfortunately" won't help, unfortunately ;) Hoping to avoid direct translations that could be troublesome.